### PR TITLE
[enterprise-4.11] OSDOCS-3756: RN entry for OSDOCS-3621

### PR DESCRIPTION
**Pull request by jldohmann**
Version(s): 4.11

Issue: https://issues.redhat.com/browse/OSDOCS-3756

Link to docs preview: http://file.rdu.redhat.com/jdohmann/OSDOCS-3621-RN/release_notes/ocp-4-11-release-notes.html#ocp-4-11-routes-spec-subdomain-field

Additional information: RN entry for #46893

### CODE REVIEW

**Comment by jldohmann** [comment]
@melvinjoseph86 could you PTAL for QE? This is the RN entry for [NE-700](https://issues.redhat.com/browse/NE-700). Thanks!

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -163,6 +163,13 @@ As a cluster administrator, you can configure `internalTrafficPolicy=Local` on a
 
 For more information, see link:https://kubernetes.io/docs/concepts/services-networking/service-traffic-policy[Service Internal Traffic Policy].
 
+[id="ocp-4-11-routes-spec-subdomain-field"]
+==== Enhancement to the Route API
+
+Previously, there was no way to specify the subdomain of a Route, and the `spec.host` field was required to set the host name. Now, you can specify the `spec.subdomain` field and omit the `spec.host` field of a Route. The router deployment that exposes the Route will use the `spec.subdomain` value to determine the host name.
```
Comment (darshan-nagaraj) [inline release_notes/ocp-4-11-release-notes.adoc:169]:
```suggestion
Previously, you could not specify the subdomain of a route, and the `spec.host` field was required to set the host name. You can now specify the `spec.subdomain` field and omit the `spec.host` field of a route. The router deployment that exposes the route will use the `spec.subdomain` value to determine the host name.
```

**Review by darshan-nagaraj — COMMENTED** [review]
Few nits inline, otherwise LGTM.

**Comment by jldohmann** [comment]
dbcd17e..92f5ef7 implement peer review feedback

**Review by kelbrown20 — COMMENTED** [review]
LGTM

**Pull request merged**